docs/02-admin/04-running-mbin/05-cli.md

```diff
@@ -121,6 +121,24 @@ Options:
 > [!NOTE] 
 > If neither `--activate` nor `--deactivate` are provided, the current verification status will be returned
 
+
+### Rotate users private keys
+
+> [!WARNING]
+> You should inform your users about this when you are running this command
```

Comment:
Maybe explain why you should tell it to the users. What is the impact, etc. Do they need to take some actions?..